product_docs/docs/pgd/5/terminology.mdx

```diff
@@ -2,83 +2,120 @@
 
 #### Eventual consistency
 
-A distributed computing consistency model stating changes to the same item in different cluster members will converge to the same value. With PGD, this is achieved through asynchronous logical replication with conflict resolution and conflict-free replicated data types.
+A distributed computing consistency model stating changes to the same item in different cluster members will eventually converge to the same value. Asynchronous logical replication with conflict resolution and conflict-free replicated data types exhibit eventual consistency in PGD.
 
 #### Failover
 
-The automated process that recognizes a failure in a highly available database cluster and takes action to connect the application to another active database. The goal is to minimize downtime and data loss.
+The automated process that recognizes a failure in a highly available database cluster and takes action to maintain consistency and availability. The goal is to minimize downtime and data loss.
 
-#### Horizontal scaling or scale out
+#### Group commit
 
-A modern distributed computing approach that manages workloads across multiple nodes, such as scaling out a web server to handle increased traffic.
+A synchronous commit mode which requires more than one PGD node to successfully receive and confirm a transaction at commit time.
 
-#### Logical replication
+#### Immediate consistency
+
+A distributed computing model where all replicas are updated synchronously and simultaneously. This ensures that all reads after a write has completed will see the same value on all nodes. The downside of this approach is its negative impact on performance.
 
-Provides more flexibility than physical replication in terms of selecting the data replicated between databases in a cluster. Also important is that cluster members can be on different versions of the database software.
+#### Kubernetes clusters
 
-#### Nines
+A Kubernetes cluster is a group of machines that work together to run containerized applications. A [PGD Cluster](pgd-cluster) can be configured to run as containerized components on a Kubernetes Cluster.
 
-A measure of availability expressed as a percentage of uptime in a given year. Three nines (99.9%) allows for 43.83 minutes of downtime per month. Four nines (99.99%) allows for 4.38 minutes of downtime per month. Five nines (99.999%) allows for 26.3 seconds of downtime per month.
+#### Logical replication
+
+A more efficient method of replicating changes in the database. Rather than duplicate the originating database’s disk blocks, logical replication instead sees the DML commands - inserts, deletes and updates,- published to all systems that have subscribed to see the changes. Each subscriber then applies the changes locally. Logical replication is not able to support most DDL
```

Comment:
@djw-m "Rather than duplicate the originating database’s disk blocks like in physical streaming replication, ..."

Reply:
```suggestion
A more efficient method of replicating changes in the database. While physical streaming replication duplicate the originating database’s disk blocks, logical replication instead sees the DML commands - inserts, deletes and updates,- published to all systems that have subscribed to see the changes. Each subscriber then applies the changes locally. Logical replication is not able to support most DDL
```